tests/native_types.rs

```diff
@@ -0,0 +1,297 @@
+extern crate cdrs;
+extern crate uuid;
+extern crate time;
+extern crate regex;
+
+mod common;
+
+use common::*;
+
+use uuid::Uuid;
+use cdrs::query::QueryBuilder;
+use cdrs::types::IntoRustByName;
+use cdrs::types::value::{Value, Bytes};
+
+use std::str::FromStr;
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr};
+
+#[test]
+fn string() {
+    let cql = "CREATE TABLE IF NOT EXISTS cdrs_test.test_string \
+               (my_ascii ascii PRIMARY KEY, my_text text, my_varchar varchar)";
+    let mut session = setup(cql).expect("setup");
+
+    let my_ascii = "my_ascii".to_string();
+    let my_text = "my_text".to_string();
+    let my_varchar = "my_varchar".to_string();
+    let values: Vec<Value> = vec![my_ascii.clone().into(),
+                                  my_text.clone().into(),
+                                  my_varchar.clone().into()];
+
+    let cql = "INSERT INTO cdrs_test.test_string \
+               (my_ascii, my_text, my_varchar) VALUES (?, ?, ?)";
+    let query = QueryBuilder::new(cql).values(values).finalize();
+    session.query(query, false, false).expect("insert");
+
+    let cql = "SELECT * FROM cdrs_test.test_string";
+    let query = QueryBuilder::new(cql).finalize();
+    let rows = session
+        .query(query, false, false)
+        .expect("query")
+        .get_body()
+        .expect("get body")
+        .into_rows()
+        .expect("into rows");
+    assert_eq!(rows.len(), 1);
+    for row in rows {
+        let my_ascii_row: String = row.get_r_by_name("my_ascii").expect("my_ascii");
+        let my_text_row: String = row.get_r_by_name("my_text").expect("my_text");
+        let my_varchar_row: String = row.get_r_by_name("my_varchar").expect("my_varchar");
+        assert_eq!(my_ascii_row, my_ascii);
+        assert_eq!(my_text_row, my_text);
+        assert_eq!(my_varchar_row, my_varchar);
+    }
+}
+
+// TODO counter, varint
+#[test]
+fn integer() {
+    let cql = "CREATE TABLE IF NOT EXISTS cdrs_test.test_integer \
+               (my_bigint bigint PRIMARY KEY, my_int int, my_boolean boolean)";
+    let mut session = setup(cql).expect("setup");
+
+    let my_bigint: i64 = 10_000_000_000_000_000;
+    let my_int: i32 = 100_000_000;
+    let my_boolean: bool = true;
+    let values: Vec<Value> = vec![my_bigint.into(), my_int.into(), my_boolean.into()];
+
+    let cql = "INSERT INTO cdrs_test.test_integer \
+               (my_bigint, my_int, my_boolean) VALUES (?, ?, ?)";
+    let query = QueryBuilder::new(cql).values(values).finalize();
+    session.query(query, false, false).expect("insert");
+
+    let cql = "SELECT * FROM cdrs_test.test_integer";
+    let query = QueryBuilder::new(cql).finalize();
+    let rows = session
+        .query(query, false, false)
+        .expect("query")
+        .get_body()
+        .expect("get body")
+        .into_rows()
+        .expect("into rows");
+    assert_eq!(rows.len(), 1);
+    for row in rows {
```

Comment:
Please add an empty row upfront the `assert_eq!` as a separator between query execution and checking result sections